### DISCUSSION TITLE
Error in deploying Azure Landing Template .json and parameters as a Custom Template

### DISCUSSION THREAD

**Discussion by neeravsalaria** [Azure/Enterprise-Scale#1229] [Q&A]
When i try to deploy template.json and parameters files as a Custom Template in azure i get the following error : 

Deployment template validation failed: 'The template variable 'deploymentUris' is not valid: The language expression property 'templateLink' doesn't exist, available properties are 'template, templateHash, parameters, mode, debugSetting, provisioningState'.. Please see https://aka.ms/arm-template-expressions for usage details.'. (Code: InvalidTemplate)

Any particular reason for it.

**Comment by jtracey93** [comment]
Hey @neeravsalaria can you tell us the files you are using and also why you are not using the portal experience (https://aka.ms/alz/portal)?

If you want to use IaC we would suggest using our [Bicep](https://aka.ms/alz/bicep) or [Terraform](https://aka.ms/alz/tf) modules instead

**Reply by neeravsalaria** [reply]
After a successful deployment using portal, i exported the template for quick landing zone creations in my other tenants. 
So i am using template.json and parameters file and trying to deploy them as custom templates and but it didnt work and fails on validation

Though when i use the option of [Azure landing zone] in Custom Deployment and just pass the parameters file, it works fine. 
So i was just curious as what is the reason behind the failure.
